fw/ctrl/Makefile

```diff
@@ -29,17 +29,11 @@ RANLIB  = $(CROSS_COMPILE)ranlib
 
 
 # flags
-CFLAGS +=   -W -Winline -std=gnu99 -g \
-            -Os \
-            -fomit-frame-pointer -ffreestanding -static -flto -fwhole-program \
-            -fno-strict-aliasing \
-            -fno-builtin -nostdlib \
-            -Wl,--relax \
-            -msoft-mul -msoft-div
-# -fpack_struct
-# -Wpacked -Wpadded
-
-LDFLAGS +=
+CFLAGS  =   -Winline -std=gnu99 -g -Os -fcommon -fomit-frame-pointer  \
+            -ffreestanding -static -flto -fno-strict-aliasing         \
+            -U_FORTIFY_SOURCE -fno-stack-protector -fno-builtin       \
+            -nostdlib -Wl,--relax -msoft-mul -msoft-div
+LDFLAGS  =
```

Comment:
`-fwhole-program` was causing a linking error I didn't properly investigate, but gains little here. *SSP* and *Fortify* were disabled due to adding (unnecessary) extra code.  `-fcommon` added due to an error on my friends system regarding multiple definitions of `file`, which, strangely, I didn't encounter.